robot/basestation/stream_capture.py

```diff
@@ -41,14 +42,14 @@ def start_recording_feed(stream_url):
     stream_is_connected = is_stream_connected(stream_url)
 
     formatted_date = datetime.datetime.now().strftime("%Y_%m_%d_%I_%M_%S")
-    filename = get_stream_shortname(stream_url) + '_' + formatted_date + ".mp4"
-    filename = save_video_dir + '/' + filename
+    video_filename = get_stream_shortname(stream_url) + '_' + formatted_date + ".mp4"
+    video_filename = save_video_dir + '/' + video_filename
 
     if stream_is_connected:
         print("Started recording " + stream_shortname)
-        active_recordings.append(stream_shortname)
-        threading.Thread(target = start_ffmpeg_record, args = (stream_url, filename)).start() 
-        return True, "Successfully started recording " + stream_shortname + " stream at " + os.path.abspath(filename)
+        active_recordings[stream_shortname] = [video_filename]
```

Comment:
Is it necessary to assign it as a array? In other words can we remove the square brackets on the right hand side and it would still work? If so let's do that.

Reply:
You know what, I was texting some explanation of why when I realized how bad design was that lol. I am changing that. I will not use the list